Governance.md

```diff
@@ -0,0 +1,78 @@
+# Governance of the rules_scala
+
+These bazel rules for scala are under the bazelbuild org but governed by a
+contributors from the community.
+
+## Goals of this document
+
+Clarify:
+0. values of the project.
+1. how to get a PR merged
+2. how long you should need to wait for a review
+3. how to resolve conflicts in project direction
+4. how to make larger and disruptive changes
+
+## Values of rules_scala
+
+1. Correctness: bazel values fully reproducible builds which are reliable and hermetic (only depend on the current state of your repository)
+2. Speed: we want to build as fast as possible. In a large repository, we believe caching and remote parallel builds are the only way to achieve this. Without correctness, we can't cache and use remote builds, so correctness enables speed. We will almost never trade off correctness for speed.
+3. Usability: we want it to be as easy as possible to adopt and to maintain a build with these rules. Usability is a lower priority than correctness or speed. We will virtually never trade either correctness or speed for more usability. Usability includes stability: we seek to absolutely minimize breaking changes. We will prefer to keep some warts or deprecated methods than force users to constantly make changes to keep their builds on the latest version of the rules.
```

Comment:
How about:
> 3. Usability: these rules strive to be friendly for adoption and friendly for the end user. To this end, we aim to be consistent with rules across the Bazel community while minimizing breaking changes. Any breaking changes will be happen after a deprecation cycle (when possible) and will be accompanied by migration guides (documentation and buildozer rewrites). Additionally, we hope to support the full breadth of the Scala ecosystem (JVM, JS, etc), but we currently give priority to usability and support of large scale JVM codebases.

Reply:
+1 on formalizing a deprecation cycle - I've had a few follow up PRs in the past to remove deprecated things, perhaps it'd be good to formalize a process (side note: a PR removing deprecated things is low hanging fruit that could be good for new contributors) but definitely bias towards stability to give users ample time to migrate and provide guides or tools to do so.

Reply:
+1 for (long, very long?) deprecation cycle but -1 to the other parts of this suggestion

Reply:
What specific issues do you have with the other parts of this suggestion?

Reply:
Mainly the removal of "Usability is a lower priority" and the breadth part. I think I've laid out my reasons in other threads dedicated to this.